src/Corrade/Utility/TypeTraits.h

```diff
@@ -163,4 +163,10 @@ template<class T> using IsStringLike = std::integral_constant<bool,
 
 }}
 
+/**
+@brief Like @cpp std::declval @ce, but declares an lvalue reference
+*/
+template<typename T>
+typename std::add_lvalue_reference<T>::type DeclareLvalueReference() noexcept;
```

Comment:
Minor: I'm usually not putting a newline after `template<..>`, keeping the whole signature a single line. (Or generally not adding a newline unless really needed to keep the code vertically terse.)